### ISSUE TITLE
Access Is Denied

### ISSUE THREAD

**Issue by darvie** [ethereum-mining/ethminer#2109]
My system keeps giving me an error message saying Access is denied I don't know why. If it helps this is my system build:
1. MSI GeForce GTX 1660 DirectX 12 GTX 1660 VENTUS XS 6G OC 6GB 192-Bit GDDR5 PCI Express 3.0 x16 HDCP Ready Video Card
2. G.SKILL Ripjaws V Series 32GB (2 x 16GB) 288-Pin DDR4 SDRAM DDR4 3200 (PC4 25600) Desktop Memory Model F4-3200C16D-32GVK
3.WD Blue 3D NAND 1TB Internal SSD - SATA III 6Gb/s M.2 2280 Solid State Drive - WDS100T2B0B
4. MSI B450 GAMING PRO CARBON MAX WIFI AM4 AMD B450 SATA 6Gb/s ATX AMD Motherboard
5. AMD RYZEN 7 3800X 8-Core 3.9 GHz (4.5 GHz Max Boost) Socket AM4 105W 100-100000025BOX Desktop Processor
6. Thermaltake Toughpower Grand RGB 650W Smart Zero Fan SLI/CrossFire Ready Continuous Power ATX12V v2.4 / EPS v2.92 80

**Comment by JayParikh20** [comment]
@darvie  did you get any solution?

**Comment by darvie** [comment]
No. I couldn’t forgive it out. I tried disabling windows defender and
firewall as well but didn’t work.

**Comment by Gimel-ai** [comment]
You can check my compiled repo with RTX 30 series and cuda 11.1 support here - https://github.com/technopremium/ethminer_RTX30series

**Comment by jean-m-cyr** [comment]
NOTE: This above recommended repo violates the terms of GPL V3 licensing and is not recommended. It is a directly derived from Ethminer but publishes no source code as required.

Try this one instead: https://github.com/no-fee-ethereum-mining/nsfminer.git
